joomla/joomla-cms
# Correct the info block handling for the category blog

Pull Request for Issue #11882

### Summary of Changes

Correct the info block handling for the category blog

### Testing Instructions

Set Position of Article Info to Below. Go to the website and you'll see the Tags under the title under the title. This happens when you choose Category Blog as Menu Item Type. The Tags are Below when you use the Featured articles as type.

### Expected result

The Tags should be Below the article in Category Blog.

### Actual result

The are Above.

### Documentation Changes Required

none.
